### PR TITLE
Add Haravan detection

### PR DESCRIPTION
**Pull request by nvanthao**
Hi maintainer,

I've just added description for Haravan, a popular ecommerce platform in Vietnam.
The website is https://www.haravan.com/
List of websites to test.
https://www.haravan.com/pages/examples

Many thanks for the awesome tool!

Cheers,

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -3352,6 +3352,15 @@
 			"implies": "OmniTouch 8660 My Teamwork",
 			"website": "http://example.com"
 		},
+		"Haravan": {
+			"cats": [
+				"6"
+			],
+			"env": "^Haravan$",
+			"icon": "Haravan.png",
+			"script": "haravan.*\\.js",
```
Comment (jvoisin) [inline src/apps.json]: Please use `"script": "/harvan\.(?:min\.)js$"` if possible.
The current rule will likely trigger false positives :)

Comment (nvanthao) [inline src/apps.json]: Hi @jvoisin,

Thanks for reviewing the pull request.

We can't use above Regex as **(?:min\.)** is not mandatory Haravan platform source code.
E.g.
`<script type="text/javascript" async="" src="https://inapp.haravan.com/buyxgety/scripts/buyxgetone/buyxgetone.js"></script>`

Cheers,

Comment (gadcam) [inline src/apps.json]: Hi @nvanthao,
Why don't try to match something like  <code>haravan_common\\.js</code> or <code>haravan\\.plugin\\.[\d.]+\\.js</code> ?
I didn't find an example that won't match with the second pattern if there is something to match in the script field.
Cheers

Comment (nvanthao) [inline src/apps.json]: Hi @gadcam,

I found this website http://blook.com.vn/ which does not match any pattern mentioned above.

It only has
`https://onapp.haravan.com/facebookinbox/static/javascripts/fb-box.js?v=1484129331617`

Cheers,

Comment (nvanthao) [inline src/apps.json]: Hi guys,

May I know if the pull request is approved?

Cheers,

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**